File: VideoEidtexter/src/main/java/com/cgfay/cameralibrary/media/combine/IAudioVideo.java
package com.cgfay.cameralibrary.media.combine;

import java.io.IOException;

/**
 * @author Harrison 唐广森
 * @description:
 * @date :2019/4/22 17:02
 */
public interface IAudioVideo {
    //初始化
    void init() throws IOException;

    //开始合并
    void startCombine();

    void endCombine();
}

File: videov2/src/main/java/com/owoh/video/media/combine/IAudioVideo.kt
package com.owoh.video.media.combine

import java.io.IOException

/**
 * @author Harrison 唐广森
 * @description:
 * @date :2019/4/22 17:02
 */
interface IAudioVideo {
    //初始化
    @Throws(IOException::class)
    fun init1()

    /**
     * 合并前做些准备
     */
    fun startCombinePrepare()

    //开始合并
    fun startCombine()

    fun endCombine()
}
